Add Location & Event Description (#466)
kubernetes/contributor-site: content/en/events/2024/kcseu/_index.md

### About

> The European summit will be a full contributor summit event taking place on March the 19th in Paris, France

Reviewer: ```suggestion
The European edition of the Contributor Summit will take place on March 19th in Paris, France
```

alongside
<a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" rel="noopener noreferrer" target="_blank">KubeCon + CloudNativeCon Europe 2024</a>
at the
<a href="https://www.viparis.com/en/our-venues/paris-convention-centre-en/access" rel="noopener noreferrer" target="_blank">Porte de Paris - Paris Convention Center</a>.

The Summit will be a healthy mix of [submitted talks](/events/2024/kcseu/schedule/#cfp),
several hours of unconference sessions, potential SIG and WG meetings, and a documentation
sprint.  This will be followed by a [Contributor Social](/events/2024/kcseu/social) with games
and refreshments.

This page will be updated as more activities are scheduled. If you have
questions or comments, please reach out to the Summit Staff by emailing
summit-team@kubernetes.io or asking in the
<a href="https://kubernetes.slack.com/messages/contributor-summit" rel="noopener noreferrer" target="_blank">#contributor-summit</a>

> slack channel.

Reviewer: ```suggestion
Slack channel.
```
